src/stravalib/model.py

```diff
@@ -769,23 +772,24 @@ class ActivityPhoto(BaseModel):
     endpoint to retrieve it
     """
 
-    athlete_id: Optional[int] = None
     activity_id: Optional[int] = None
     activity_name: Optional[str] = None
-    ref: Optional[str] = None
-    uid: Optional[str] = None
-    unique_id: Optional[str] = None
+    athlete_id: Optional[int] = None
     caption: Optional[str] = None
-    type: Optional[str] = None
-    uploaded_at: Optional[datetime] = None
     created_at: Optional[datetime] = None
     created_at_local: Optional[datetime] = None
+    default_photo: Optional[bool] = None
     location: Optional[LatLon] = None
-    urls: Optional[dict[str, str]] = None
-    sizes: Optional[dict[str, Sequence[int]]] = None
     post_id: Optional[int] = None
-    default_photo: Optional[bool] = None
+    sizes: Optional[dict[str, Sequence[int]]] = None
     source: Optional[int] = None
+    type: Optional[int] = None
+    unique_id: Optional[str] = None
+    uploaded_at: Optional[datetime] = None
+    urls: Optional[dict[str, AnyHttpUrl]] = None
+
+    # ref: Optional[str] = None
+    # uid: Optional[str] = None
 
```

Comment:
```suggestion
```

Reply:
If you agree i didn't see these in the actual return.

Reply:
Note - i did notice after another pass that these attrs are used below. i can revisit if i'm missing something (which i think i must be).